doc/introduction/interfaces.rst

```diff
@@ -73,6 +73,15 @@ See the links below for walkthroughs of each specific interface:
                     </div>
                 </a>
             </div>
+            <div class="col-lg-4 mb-2 align-items-stretch">
```

Comment:
```suggestion
            <div class="col-lg-3 mb-2 align-items-stretch">
```

Reply:
The ones above also need to be changed to `col-lg-3`. Bootstrap has a 12-width column, so this will allow all logos to fit in a single row:

![image](https://user-images.githubusercontent.com/2959003/103986451-b4e13f80-51c5-11eb-8a9b-869da367fbc7.png)